src/implementations/twig/components/tabs/tabs.html.twig

```diff
@@ -33,7 +33,8 @@
 {% set _css_class = 'ecl-tabs' %}
 {% set _extra_attributes = [
```

Comment:
i see that we normally define our internal variable for attributes as a string that gets populated by looping into the extra_attributes param, and it's ok since it's going to be rendered as a string. But when we have "internal" attributes to set, it would feel natural to merge these values into the extra_attributes array and get the _extra_attributes string as the result of the internal and external attributes processing.

Reply:
Not sure, I properly understand the issue here, we are doing same in expandable and news-ticker